### PR TITLE
Fix typos + better highlighting + details on some proposals

### PR DESCRIPTION
**Pull request by sarupbanskota**
| Proposal                                                            | PR      | Label  |
|---------------------------------------------------------------------|---------|--------|
| [Rest/Spread Properties](#restspread-properties)                    | no      | yes    |
| [Asynchronous Iteration](#asynchronous-iteration)                   | no      | no     |
| [Dynamic Import](#dynamic-import)                                   | no      | no     |
| [RegExp Unicode Property Escapes](#regexp-unicode-property-escapes) | no      | no     |
| [RegExp Named Capture Groups](#regexp-named-capture-groups)         | no      | no     |
| [RegExp DotAll Flag](#regexp-dotall-flag)                           | no      | no     |
| [`function.sent`](#functionsent)                                    | no      | no     |
| [Class Fields](#class-fields)                                       | yes     | yes    |
| [Class and Property Decorators](#class-and-property-decorators)     | yes     | yes    |
| [BigInt](#bigint)                                                   | yes     | yes    |
| [`import.meta`](#importmeta)                                        | yes     | yes    |
| [`export * as ns from "mod";`](#export-extensions)                  | no      | no     |
| [`export v from "mod";`](#export-extensions)                        | no      | no     |
| [Generator Arrow Functions](#generator-arrow-functions)             | no      | no     |
| [Optional Chaining](#optional-chaining)                             | no      | no     |
| [`do` Expressions](#do-expressions)                                 | no      | no     |
| [Numeric Separator](#numeric-separator)                             | yes     | yes    |
| [Function Bind](#function-bind)                                     | no      | no     |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,8 +319,9 @@ func.call(obj, val)
 **Babylon Label**: [Spec: BigInt](https://github.com/babel/babylon/issues?utf8=%E2%9C%93&q=label%3A%22Spec%3A%20BigInt%22%20)  
 <details>
 <summary>Code Example</summary>
-```js
-1n; // integer
+
+```
```
Comment (Kovensky) [inline README.md]: I wonder if these should be left as `js` despite the "syntax errors"... 🤔

Comment (sarupbanskota) [inline README.md]: @Kovensky yeah prob a good idea, I initially removed it cuz I incorrectly assumed the `js` to be responsible for broken formatting, but it was the absence of newline

Comment (sarupbanskota) [inline README.md]: @Kovensky added them back. 

I don't have push rights, so if you're happy with what's there so far, and don't mind, would you please merge this PR? (and i'll do another batch in a different one). That way I don't need to repeatedly rebase if somebody makes updates to `master`! :)

**Comment by hzoo** [comment]
Thanks @sarupbanskota!

**Pull request merged**